ant/src/balances.lua

```diff
@@ -0,0 +1,50 @@
+local utils = require(".utils")
+local json = require(".json")
+
+Balances = Balances or {}
+
+local balances = {}
+
+function balances.walletHasSufficientBalance(wallet)
+	return Balances[wallet] ~= nil and Balances[wallet] > 0
+end
+
+function balances.info()
+	utils.reply(json.encode({
+		Name = Name,
+		Ticker = Ticker,
+		TotalSupply = 1,
+		Logo = Logo,
+		Denomination = Denomination,
+	}))
+end
+
+function balances.transfer(to)
```

Comment:
The transfer needs to do more than this.

It needs to set balances, it needs to set ownership and it needs to empty Controllers

Balances = { [to] = 1 }
Owner = to
Controllers = {}